packages/docs/docs/zh/api/cli.md

```diff
@@ -40,7 +40,7 @@ vuepress <command> targetDir [options]
 当服务端准备就绪时自动打开浏览器。
 
 ### --no-clear-screen
-当 dev server 就绪时不清除屏幕。
+当 dev server 就绪时不清除屏幕。请注意dev server不會在调试模式下清除屏幕。
```

Comment:
```suggestion
当 dev server 就绪时不清除屏幕。请注意 dev server 不会在调试模式下清除屏幕。
```
Add spaces between English and Chinese, and use Simplified Chinese

Reply:
Thanks for catching.